Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+             
+             }
+
+            .name {
+             color: dodgerblue;
+             font-size: 30px;
+             font-family: Helvetica;
+             
+             }
+            .name2 {
+             color: plum;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name3 {
+             color:orange;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+             .name4 {
+             color: red;
+             font-size: 30px;
+             font-family: Helvetica;
+             }
+
+             img.rounded-corners {
+border-radius: 15px;
+}
+
+             .image-container {
+  display: flex;
+  justify-content: center;
+}
+             
+        
+            
+            </style>
+    </head>
+        <body>
+                
+                <!-- get the header image to center on the page -->
+                <div class="image-container"><img src="TMNTHead.png" alt="TMNTHead" style="width:1400px;height:832px;"></div>
+            
+            <div class="padded">
+            <!-- see if you can get this a little bit narrower so that the text wraps to more lines to match the design a little closer -->
+            
+            <p style="font-family:Helvetica;">
+
+            <div id="rcorners2"><img src="leonardo.gif" alt="Leonardo" class="rounded-corners" style="float:left;width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name"><b>Leonardo</b></span>
+            <hr style="height:0px;border-top: 3px solid dodgerblue" />
+
+            <span class="main">The tactical, courageous leader and devoted student of his sensei,
+            Leonardo wears a blue mask and wields two katana. As the most conscientious
+            of the four, he often bears the burden of responsibilty for his brothers,
+            which commonly leads to conflict with Raphael.
+            <br>
+            <br>
+            Leonardo was named after the Italian polymath, painter, engineer,
+            inventor, writer, anatomist, and sculptor <a href="https://en.wikipedia.org/wiki/Leonardo_da_Vinci" style="color:dodgerblue">Leonardo da Vinci.</a></span></div>
+
+        
+            <!-- instead of using br tags, try adding padding to the bottom of each section. This is easiest accomplished by wrapping each section in its div tag like htis:
+                <div style="padding-bottom:20px;"> <-- section
+                        <div></div> <-- image
+                        headline
+                        text
+                </div>
+        -->
+            <div id="rcorners3"><img src="donatello.gif" alt="Donatello" class="rounded-corners" style="float:right;width:200px;height:200px;">
+            </div>
+
+            <div style="padding-bottom:100px;"><span class="name2"><b>Donatello</b></span>
+            <hr style="height:0px;border-top: 3px solid plum" />
```

Comment:
Can you make a custom class to share this style with all hr tags?

```
hr.something {
    height: 0;
    padding-bottom: 100px;
    border-top: #DDA0DD;
}
```

`<hr class="something">`